setup.py

```diff
@@ -78,7 +170,7 @@ def load_long_description():
 # the goal of the project is simplicity for researchers, don't want to add too much
 # engineer specific practices
 setup(
-    name='pytorch-lightning',
+    name="pytorch-lightning-nightly",
```

Comment:
Not sure we want to change name here.

Reply:
i didn't change it, or at least not on purpose I will roll back this line